### PR TITLE
FRW-1204 Integrated propel changes and Symfony 6x

### PR DESCRIPTION
**Pull request by olhalivitchuk**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/FRW-1204

### CODE REVIEW

**Review by yaroslav-spryker — APPROVED** [review]

File: config/Shared/config_default.php [hunk 1]
```diff
@@ -335,6 +336,9 @@
 $config[PropelConstants::ZED_DB_REPLICAS] = json_decode(getenv('SPRYKER_DB_REPLICAS') ?: '[]', true);
 $config[PropelConstants::USE_SUDO_TO_MANAGE_DATABASE] = false;
 
+// >>> DATABASE REPLICA CACHE
+$config[PropelReplicationCacheConstants::CACHE_TTL] = 2;
```
Comment (asmarovydlo) [inline config/Shared/config_default.php:340]: Shouldn't we also integrate 
```
$config[PropelReplicationCacheConstants::IS_REPLICATION_ENABLED] = (bool)$config[PropelConstants::ZED_DB_REPLICAS];
```

From https://github.com/spryker/suite-nonsplit/pull/6379/files

Comment (olhalivitchuk) [reply config/Shared/config_default.php:340]: It was, but we agreed the replication has to be disabled by default and I removed it. 
Please, check https://spryker.slack.com/archives/C03KPK97UH3/p1683029569719389

**Review by asmarovydlo — APPROVED** [review]

**Review by asmarovydlo — COMMENTED** [review]
1. Why we updated so huge module set?
2. GithubActions is red because of the modules that we've updated

**Comment by olhalivitchuk** [comment]
1. > Why we updated so huge module set?
Because of dependency on `spryker/locale: ^4.0.0`, `spryker/currency`:"^4.0.0" in modules from other features and from root packages, we can't just update `spryker-feature/spryker-core`

2. I'll fix it, but looks like it came from `internal` branch after merge I had to do it because of conflicts

**Comment by asmarovydlo** [comment]
> 1. > Why we updated so huge module set?
> 
> Because of dependency on `spryker/locale: ^4.0.0`, `spryker/currency`:"^4.0.0" in modules from other features and from root packages, we can't just update `spryker-feature/spryker-core`
> 
> 2. I'll fix it, but looks like it came from `internal` branch after merge I had to do it because of conflicts

1. Ok, got it, thanks
2. Please fix it.

**Review by dimitriyTsemma — APPROVED** [review]

**Review by gechetspr — APPROVED** [review]

**Pull request merged**